### PR TITLE
Enable Log Analytics for logging_project_bucket_config resource

### PR DESCRIPTION
**Pull request by ehorning**
Enable Log Analytics for project logging bucket resource

Fixes https://github.com/hashicorp/terraform-provider-google/issues/13934

If this PR is for Terraform, I acknowledge that I have:

- [x] Searched through the [issue tracker](https://github.com/hashicorp/terraform-provider-google/issues) for an open issue that this either resolves or contributes to, commented on it to claim it, and written "fixes {url}" or "part of {url}" in this PR description. If there were no relevant open issues, I opened one and commented that I would like to work on it (not necessary for very small changes).
- [x] Ensured that all new fields I added that can be set by a user appear in at least one [example](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/templates/terraform/examples) (for generated resources) or [third_party test](https://github.com/GoogleCloudPlatform/magic-modules/tree/main/mmv1/third_party/terraform/tests) (for handwritten resources or update tests).
- [x] [Generated Terraform providers](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/generate-providers.md), and ran [`make test` and `make lint`](https://googlecloudplatform.github.io/magic-modules/docs/getting-started/run-provider-tests/#run-unit-tests) in the generated providers to ensure it passes unit and linter tests.
- [x] [Ran](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/docs/content/docs/getting-started/run-provider-tests.md) relevant acceptance tests using my own Google Cloud project and credentials (If the acceptance tests do not yet pass or you are unable to run them, please let your reviewer know).
- [x] Read the [Release Notes Guide](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/.ci/RELEASE_NOTES_GUIDE.md) before writing my release note below.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:enhancement
logging: added `enable_analytics` field to `google_logging_project_bucket_config`
```

### CODE REVIEW

**Comment by ehorning** [comment]
>

**Comment by ehorning** [comment]
> /gcbrun

@shuyama1 what does this mean?

**Comment by shuyama1** [comment]
@ehorning would you mind rebasing your PR? Thanks!

**Comment by ehorning** [comment]
> @ehorning would you mind rebasing your PR? Thanks!

@shuyama1 done

**Comment by shuyama1** [comment]
tpg and tpgb tests failed due to a linter problem. You may want to run `gofmt -w mmv1/third_party/terraform/resources/resource_logging_project_bucket_config.go`

**Comment by ehorning** [comment]
> tpg and tpgb tests failed due to a linter problem. You may want to run `gofmt -w mmv1/third_party/terraform/resources/resource_logging_project_bucket_config.go`

fixed

File: mmv1/third_party/terraform/website/docs/r/logging_project_bucket_config.html.markdown [hunk 1]
```diff
@@ -95,6 +107,8 @@ The following arguments are supported:
 
 * `retention_days` - (Optional) Logs will be retained by default for this amount of time, after which they will automatically be deleted. The minimum retention period is 1 day. If this value is set to zero at bucket creation time, the default time of 30 days will be used.
 
+* `enable_analytics` - (Optional) Whether or not Log Analytics is enabled. Logs for buckets with Log Analytics enabled can be queried in the **Log Analytics** page using SQL queries.
```
[the remark this answers is not in the document]
Comment (ehorning) [reply mmv1/third_party/terraform/website/docs/r/logging_project_bucket_config.html.markdown]: done

**Review by shuyama1 — APPROVED** [review]
Thank you!

**Comment by ehorning** [comment]
> Thank you!

Thanks for the review/help!

**Pull request merged**